readme.md

```diff
@@ -69,6 +69,8 @@ Watch task helps you reduce your efforts during development extensions. If the t
 gulp watch
 ```
 
+In order for Live-reload to work, reload extension manually after starting `gulp watch`.
```

Comment:
In case when extension is loaded and I'm using chrome. Then I want to continue working on extension and start `gulp watch`, in that case extension needs to be reloaded.